docs/fleet/cli.md

```diff
@@ -702,19 +702,103 @@ You can use the `module build start` or `module build local` commands to build y
 - Use `build start` to build or compile your module on a cloud build host that might offer additional platform support than you have access to locally.
 - Use `build local` to quickly test that your module builds or compiles as expected on your local hardware.
 
-To configure your module's build steps, add a `build` object to your [`meta.json` file](#the-metajson-file), including the following:
+To configure your module's build steps, add a `build` object to your [`meta.json` file](#the-metajson-file).
+Developers can either have a single build file for all platforms, or platform specific files:
+
+{{< tabs >}}
+{{% tab name="Single Build File" %}}
 
 ```json {class="line-numbers linkable-line-numbers"}
 "build": {
   "setup": "./setup.sh",                  // optional - command to install your build dependencies
-  "build": "make module.tar.gz",          // command that will build your module
-  "path" : "module.tar.gz",               // optional - path to your built module
+  "build": "sh build.sh",                 // command that will build your module
+  "path" : "dist/archive.tar.gz",         // optional - path to your built module
                                           // (passed to the 'viam module upload' command)
   "arch" : ["linux/amd64", "linux/arm64"] // architecture(s) to build for
 }
 ```
 
-For example, the following extends the `my-module` <file>meta.json</file> file from the previous section with a new `build` object to control its build parameters when used with `module build start` or `module build local`:
+{{%expand "Click to view example build.sh" %}}
+
+```sh { class="command-line"}
+#!/bin/bash
+set -e
+UNAME=$(uname -s)
+
+if [ "$UNAME" = "Linux" ]
+then
+    echo "Installing venv on Linux"
+    sudo apt-get install -y python3-venv
+fi
+if [ "$UNAME" = "Darwin" ]
+then
+    echo "Installing venv on Darwin"
+    brew install python3-venv
+fi
+
+python3 -m venv .venv
+. .venv/bin/activate
+pip3 install -r requirements.txt
+python3 -m PyInstaller --onefile --hidden-import="googleapiclient" src/main.py
+tar -czvf dist/archive.tar.gz dist/main
+```
+
+{{% /expand%}}
+
+{{% /tab %}}
+{{% tab name="Platform Specific" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+"build": {
+  "path" : "dist/archive.tar.gz",               // optional - path to your built module
+                                                // (passed to the 'viam module upload' command)
+  "arch": {
+        "linux/arm64": {
+          "build": "sh build-linux-arm64.sh" // command that will build your module
+        },
+        "darwin/arm64": {
+          "build": "sh build-darwin-arm64.sh" // command that will build your module
+        }
+      } // architecture(s) to build for
+}
+```
+
+{{%expand "Click to view example build-linux-arm64.sh" %}}
```

Comment:
non critical, but this `build-linux` + the `build-darwin` below could be merged -- can do the `if uname` thing from the previous script

Reply:
Commenting out this whole section re. Michael's comment, but I meant to show the separate files for this approach!